### ISSUE TITLE
For screen readers, add numbering/headings to program cards on homepage

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1522]
On the applicant homepage, they get a long list of cards, one per program. The trouble is that, if you're using a screen reader alone, it's hard to know where one card stops and where one card ends, since all the text in the cards uses the same HTML tags (namely, `<div>`s).

Two possible solutions:

1. Use a `<h4>` or something for the program names on the cards, so that screen readers can call out when a new card is being started.
1. Add a `sr-only` (screen reader only) bit of text before each program title, saying something like "Program $N of $M -- $ProgramName", e.g. "Program 5 of 13 -- Wave Internet". 

Either one should work, I think.

## Original tester's report

> ACTUAL RESULT: There is no indication to the user when entering and ending any program's card.
> EXPECTED RESULT: While navigating through the programs, the user should be informed when user is entering and ending any program card.
> Suggestion: Provide numbered grouping list for the program cards. i.e., If there are a total of 13 cards, the announcement can be 1 out of 13, 2 out of 13 to avoid confusion.
